test/integration/next-image-new/next-url-server-prefix/pages/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Image from 'next/image'
+import img from '../public/test.jpg'
+
+const Page = () => {
+  return (
+    <div>
+      <p>Asset Prefix</p>
+      <Image id="test1" src={img} placeholder="blur"></Image>
```

Comment:
```suggestion
      <Image id="test1" src={img} placeholder="blur" />
      <Image id="test2" src="/test.jpg" width="200" height"200" />
```

Lets add a second test to see what happens with the string src